81-save-an-image-locally.md

```diff
@@ -0,0 +1,12 @@
+# How To Save An Image Locally Using Python Whose URL Address I Already Know?
+
+We will use the following code to save an image locally from an URL address:
+
+```python
+import urllib.request
+urllib.request.urlretrieve("URL", "local-filename.jpg")
```

Comment:
I tried this code and it is not working for me, do you know why?
![image](https://user-images.githubusercontent.com/12773753/126868373-3aecf88a-4736-4afb-b9ec-120946e9c76c.png)
>>> urllib.request.urlretrieve("https://www.pixelstalk.net/wp-content/uploads/2016/07/HD-Peaceful-Background.jpg", "local-filename.jpg")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.8/urllib/request.py", line 247, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
  File "/usr/lib/python3.8/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.8/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.8/urllib/request.py", line 640, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.8/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.8/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden